CreateGame.py

```diff
@@ -0,0 +1,76 @@
+import logging, uuid, datetime, boto3, json
+from botocore.exceptions import ClientError
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def handler(event, context):
+    logger.info('Handling event {} - context {}', event, context)
+
+    responseMessage = event["data"]["response"]["sms"]
+    errors = event["data"]["errors"]
+
+    # Initial game state
+    game = {
+        "gameState": "over"
+    }
+
+    dynamodb = boto3.resource('dynamodb', region_name='us-east-1')
+    hangmanGame = dynamodb.Table('HangmanGame')
+
+    # Get the game from the game table
+    response = hangmanGame.scan()
+    gameExists = len(response["Items"]) == 1
+    if gameExists:
+        game = response["Items"][0]
+        logger.info('Found existing game record {}', game)
+        responseMessage = responseMessage + "\n" + "A game record exists"
+    else:
+        # generate the first and only game id. Starting a new game will reuse this uuid
+        print("Game does not exist!")
+        game["Id"] = str(uuid.uuid4())
+        logger.info('No existing game record. Creating game id {}', game["Id"])
+        responseMessage = responseMessage + "\n" + "A game record does not exist"
+
+    # If the game state is "over", update the game state to "created"
+    if game["gameState"] == "over":
+        # generate timestamp
+        game["startDatetime"] = str(datetime.datetime.utcnow().isoformat())
+
+        # set state to created
+        game["gameState"] = 'created'
```

Comment:
Can we be consistent with single/double quotes? Python doesn't care which you use but we should be consistent ourselves.